### ISSUE TITLE
Idea: Governance / Forum RSS feeds

### ISSUE THREAD

**Issue by mochet** [Joystream/pioneer#3594]
SSIA

**Comment by dmtrjsg** [comment]
@mochet could you pls elaborate on the triggers/ events that you'd benefit from the most?

**Comment by dmtrjsg** [comment]
I don;t think this would be ideal to work on without proper Backend, which is in the pipeline now
#3491
